pkg/dsl/model/instance.go

```diff
@@ -1,6 +1,12 @@
 package model
 
 import (
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"cuelang.org/go/cue/build"
+
```

Comment:
goimport

I thought the IDE does this automatically

Reply:
Do you know how to configure the IDE?